tests/provision/become/test.sh

```diff
@@ -7,7 +7,17 @@ rlJournalStart
     if [[ "$PROVISION_METHODS" =~ container ]]; then
         rlPhaseStartSetup
             rlRun "pushd data"
-            rlRun "podman build -t become-container-test:latest ."
+            # Try several times to build the container
+            # https://github.com/teemtee/tmt/issues/2063
+            for attempt in {1..5}; do
+                if podman build -t become-container-test:latest . ; then
+                    break
```

Comment:
How about `rlPass "container built"` so we have successful assert present?

Reply:
Nice idea, done in 7e87caa0.

Reply:
Hm, at last I went with a simplified version using `rlWaitForCmd`: 65571efd